tests/e2e/tests/critical-path/database-overview/database-recommendations.e2e.ts

```diff
@@ -0,0 +1,44 @@
+import { acceptLicenseTermsAndAddDatabaseApi } from '../../../helpers/database';
+import { rte } from '../../../helpers/constants';
+import {
+    commonUrl,
+    ossStandaloneConfig
+} from '../../../helpers/conf';
+import { deleteStandaloneDatabaseApi } from '../../../helpers/api/api-database';
+import {MemoryEfficiencyPage, MyRedisDatabasePage} from '../../../pageObjects';
+import {RecommendationsPage} from '../../../pageObjects/recommendations-page';
+import {Common} from '../../../helpers/common';
+
+const myRedisDatabasePage = new MyRedisDatabasePage();
+const memoryEfficiencyPage = new MemoryEfficiencyPage();
+const recommendationPage = new RecommendationsPage();
+const common = new Common();
+
+fixture `Upvote recommendations`
```

Comment:
please move this tests to tests\e2e\tests\critical-path\memory-efficiency\recommendations.e2e.ts